format_notification.py

```diff
@@ -0,0 +1,72 @@
+#!/bin/env python
+
+# this while loop is for not considering two string fields which you get right after starting a process
+# Test string ->['string "ShanonPark posted in "HEY! YOU! What are you up to right now! (Part 3)" - Linkin Park / LPU Forums"'] 2 
+
+import time
+import sys
+import notify2
+from setup import main
+
+def EditString(data: str) -> str:
+    return data[0].split('"', 1)[1].replace('"', '')
+
+def Notify(summary, data, urgency, timeout=800000):
+    notify2.init("AutoReplyToLP")
+    pop_up = notify2.Notification(summary, data)
+    pop_up.set_urgency(urgency)
+    pop_up.set_timeout(timeout)
+    return pop_up
+
+def countdown_timer(timeMS):
+    # timeMS should be in millisecond
+    minute = timeMS // 60000
+    second = 59
+    pop_up = Notify(f"{minute:02d} : {second:02d}", "Time Left", 1)
+
+    while minute > -1:
+        pop_up.update(f"Time Left -> {minute:02d} : {second:02d}")
+        time.sleep(1)
```

Comment:
This is OK for this application, but just so you know that this will take longer than a second, and each cycle will make it drift a bit more.

For accurate time before the loop do something like

start_time = Time.now()

then in the while condition

while Time.now() - start_time > running time

will give you more accurate time.